docker/test/integration/minifi/test/__init__.py

```diff
@@ -117,12 +119,15 @@ def log_nifi_output(self):
         for container in self.containers:
             container = self.client.containers.get(container.id)
             logging.info('Container logs for container \'%s\':\n%s', container.name, container.logs())
+            if b'Segmentation fault' in container.logs():
+                self.segfault=true
```

Comment:
Actually this line is wrong, this should be True (capital 't')

Reply:
Thanks!